cranelift-codegen/meta/src/gen_legalizer.rs

```diff
@@ -87,6 +87,11 @@ fn unwrap_inst(
                 } else if op.is_value() {
                     let n = inst.value_opnums.iter().position(|&i| i == op_num).unwrap();
                     fmtln!(fmt, "func.dfg.resolve_aliases(args[{}]),", n);
+                } else if op.is_varargs() {
+                    let n = inst.imm_opnums.iter().chain(inst.value_opnums.iter()).max().map(|n| n + 1).unwrap_or(0);
+                    fmtln!(fmt, "\
+                        args.iter().skip({}).map(|&arg| func.dfg.resolve_aliases(arg)).collect::<Vec<_>>(),\
+                    ", n);
```

Comment:
I'm sorry I don't have as much context here as I should; can you remind me what this patch needs varargs for?

Reply:
https://github.com/CraneStation/cranelift/pull/795/files#diff-6263bd0b84702d1f31e46a46ce47b28fR257-R291

Legalizing `brz.i128` and `brnz.i128`.